tools/autograd/templates/python_variable_methods.cpp

```diff
@@ -674,12 +678,12 @@ PyMethodDef variable_methods[] = {
   {"__truediv__", (PyCFunction)THPVariable_div, METH_VARARGS | METH_KEYWORDS, NULL},
   {"__idiv__", (PyCFunction)THPVariable_div_, METH_VARARGS | METH_KEYWORDS, NULL},
   {"__mod__", (PyCFunction)THPVariable_remainder, METH_VARARGS | METH_KEYWORDS, NULL},
-  {"__bool__", (PyCFunction)THPVariable_bool, METH_NOARGS, NULL},
+  {"__bool__", (PyCFunction)THPVariable_tensor_to_python_bool_var, METH_NOARGS, NULL},
```

Comment:
Naming: Why not `THPVariable_bool_scalar` to be consistent with the ones for float/int ?

Reply:
good catch